core/server/config/env/config.production.json

```diff
@@ -8,5 +8,10 @@
     },
     "paths": {
         "contentPath": "content/"
+    },
+    "logging": {
+        "level": "info",
+        "rotation": true,
+        "transports": ["file"]
```

Comment:
> Separate logs for errors and requests are quite common in HTTP servers (Apache, Nginx) to get a quick overview of what is going wrong when a request fails.

Yes i am aware of that. And i agree, having a separate error log file might be useful, which only contains requests with errors (or single errors outside of the HTTP scope). But in future thinking, in bigger software nobody looks at plain log files. People send their logs to centralised logging system. And with this tool they can do what they want with the JSON log files. (order, send mails when errors are critical, analyse user behaviour etc.).

> It is possible to register multiple streams with bunyan that operate on different log levels or even different serializers. If we expose this functionality in the config we could have different log files (or even different systems) with different visibility.

Yeah, we can register what we want, we just need to define what we need (or what people need). I will write up a more detailed expectation for the default logging component we will offer into #7431 . This will contain: logging config and default logging behaviour. I am happy about feedback then :)